packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -395,6 +417,23 @@ const App = (props) => {
                         </Box>
                     </CurrencyProvider>
                 </IntlProvider>
+                {ACTIVE_DATA_ENABLE && (
+                    <script
+                        type="text/javascript"
+                        src={getAssetUrl('static/dwanalytics-22.2.js')}
+                        id="dwanalytics"
+                        async="async"
+                        onLoad={trackPage}
+                    ></script>
+                )}
+                {ACTIVE_DATA_ENABLE && (
+                    <script
+                        src={getAssetUrl('static/dwac-21.7.js')}
+                        type="text/javascript"
+                        id="dwac"
+                        async="async"
+                    ></script>
+                )}
```

Comment:
Because Helmet hoists its contents into the `<head>`, that script could be brought down here and the three could be included in a single conditional:
```jsx
{ACTIVE_DATA_ENABLE && (
  <>
    <Helmet><script></Helmet>
    <script>
    <script>
  </>
)}
```

If you do that, you could also then extract it into an independent `<ActiveData>` component, so that customers can override that component without needing to override the entire `<App>` component.

Reply:
this seems to be an enhancement for future work.